docs/cloud/provision/aws/README.md

```diff
@@ -52,30 +52,28 @@ When you click on the button, you get to the cluster creation form.
 On the first part of the form you need to specify the new cluster name, the cloud provider (AWS) and the EventStoreDB version (currently it's only 20.6). Further, you need to choose the deployment size (single instance or three-node cluster) and whether to start server-side projections by default.
 
 ::: card
-
 ![AWS cluster first part](./images/aws-new-cluster-1.png)
-
 :::
 
 ::: warning Projections impact performance
 Both system projections and user-defined projections produce new events. Carefully consider the impact of enabled projections on database performance. Please refer to the [Performance impact](../../../server/5.0.8/server/projections/README.md#performance-impact) section of the projections documentation to learn more.
 :::
 
-The lower section of the form allows choosing the instance size for cluster nodes. Currently, only three instance sizes are available. The `F1` size is the lower-edge, aiming mainly to support testing scenarios and experiments due to its low price. Two other sizes - `C4` and `M4` are production-grade.
+The lower section of the form allows choosing the instance size for cluster nodes. Currently, only three instance sizes are available. The `F1` size is the lower-edge, aiming mainly to support testing scenarios and experiments due to its low price. Other instance sizes are production-grade.
 
-Moving forward, EventStoreDB Cloud would have more instance sizes available.
+::: tip Vertical scaling
+At this moment, it is not possible to change the cluster node instance size. You can still resize cluster instances by taking a backup and restoring it to a different cluster with larger or smaller instances.
```

Comment:
Is the backup strategy described in https://developers.eventstore.com/server/20.6/server/operations/database-backup.html applicable for that? If yes it might be worth adding the link.

Reply:
No, it's not applicable as customers don't have access to cluster nodes.

Reply:
So how can they do that?

Reply:
It's not a part of this page.

Reply:
I understand that it doesn't have to be explained here in details, but if we're recommending such strategy, we should (eventually) guide the user to do that - e.g. by linking to the other document.

Reply:
Of course, but, again, it's not a part of this PR.